ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1271,3 +1271,59 @@ def test_tensorflow_linspace(
         axis=axis,
         on_device=on_device,
     )
+   
+
+# tile
+@st.composite
+def _multiple_shape_helper(draw):
+    input_dtype, input_array, input_shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            ret_shape=True
+        ))
+    
+    # input = dtypes_and_input[1][0]
+    input_dims = len(input_shape)
+    
+    shape = draw(helpers.get_shape(
+        min_num_dims=1, max_num_dims=1,
+        min_dim_size=input_dims, max_dim_size=input_dims
+    ))
+    dt_n_multiples = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["int32", "int64"],
+            shape=shape))  
```

Comment:
Instead of drawing the shape separately, I think you could pass the dimension arguments straight to `dtype_and_values` here.